content/blog/2025/07/2025-07-23-postgres-corruption-postmortem.md

```diff
@@ -0,0 +1,278 @@
++++
+date = "2025-07-23"
+title = "How we discovered, and recovered from, Postgres corruption on the matrix.org homeserver"
+
+[taxonomies]
+author = ["Richard van der Hoff"]
+category = ["General"]
++++
+
+Greetings from Element's backend/SRE team, who run the matrix.org homeserver on behalf of the Matrix.org Foundation.
```

Comment:
```suggestion
Greetings from Element's backend/SRE team, who run the `matrix.org` homeserver on behalf of the Matrix.org Foundation.
```

for consistency with the rest of the post

Reply:
(you can also link to https://matrix.org/homeserver/ if needed)

Reply:
+1 to the latter